docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,102 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+* Rename configuration property used for selecting a policy for low memory query
+  killer from `query.low-memory-killer.policy` to
+  `query.low-memory-query-killer.policy`. ({issue}`12242`)
+* Introduce specific configuration property
+  `query.low-memory-task-killer.policy` for selecting a policy for low memory
+  task killer. The policy is used if task level retries are enabled
+  (`retry-policy=TASK`). ({issue}`12242`)
+* Introduce `least-wasted` low memory task killer policy. This policy avoids
+  killing tasks that are already executing for a long time, so a significant
+  amount of work is not wasted.  ({issue}`12242`)
+* Improve Advanced Encryption Standard (AES) processing performance on ARM64 processors. ({issue}`12251`)
+* Fix potential planning failure of queries with multiple subqueries.  ({issue}`12199`)
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
+* Fix regression in file-based system and connector access controls. ({issue}`12204`)
```

Comment:
"Regression" is too vague. We need to describe what problem is fixed.

Reply:
Any suggestions or insight @Praveen2112 @ksobolew ?

Reply:
Maybe something like "Fix regression in the way file-based system and connector access controls select permission entries when column masks and row filters are present." Isn't it too long? "system and connector" could probably be removed to leave just "file-based access controls".